interfaces/builtin/anbox_support.go

```diff
@@ -0,0 +1,512 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const anboxSupportSummary = `allows operating as the Anbox container management service`
+
+const anboxSupportBaseDeclarationPlugs = `
+  anbox-support:
+    allow-installation: false
+    deny-auto-connection: true
+`
+
+const anboxSupportBaseDeclarationSlots = `
+  anbox-support:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
```

Comment:
You can drop anboxSupportBaseDeclarationSlots here and below-- snaps aren't providing anything on the slot side at this time and you've defined it elsewhere as an implicit interface.